### PR TITLE
Netcore: configuration of supported image file types

### PR DESCRIPTION
**Pull request by AndyButland**
Based on a comment coming out of [this PR](https://github.com/umbraco/Umbraco-CMS/pull/8938) I've removed this list of supported image file types from configuration and set up instead as a property on `IImageUrlGenerator`, so different providers can indicate support for different sets of images.

Relates to #8651

### CODE REVIEW

File: src/Umbraco.Infrastructure/PropertyEditors/ImageCropperPropertyEditor.cs [hunk 1]
```diff
@@ -48,17 +48,17 @@ public class ImageCropperPropertyEditor : DataEditor, IMediaUrlGenerator
...
         {
             _mediaFileSystem = mediaFileSystem ?? throw new ArgumentNullException(nameof(mediaFileSystem));
             _contentSettings = contentSettings.Value ?? throw new ArgumentNullException(nameof(contentSettings));
             _dataTypeService = dataTypeService;
-            _localizationService = localizationService;
             _ioHelper = ioHelper;
 
             // TODO: inject?
-            _autoFillProperties = new UploadAutoFillProperties(_mediaFileSystem, logger, contentSettings);
+            _autoFillProperties = new UploadAutoFillProperties(_mediaFileSystem, logger, imageUrlGenerator);
```
Comment (bergmania) [inline src/Umbraco.Infrastructure/PropertyEditors/ImageCropperPropertyEditor.cs:61]: Not really related to this PR, but think it make sense to register UploadAutoFillProperties in the container and inject it instead of the arguments needed . Can be a separate PR :)

**Review by bergmania — COMMENTED** [review]
Looks good. Just a small comment..

**Comment by bergmania** [comment]
I will take a look at the conflicts and can implement my suggestions too

**Comment by bergmania** [comment]
@AndyButland I updated with my suggestions.. Please review my changes and see if you agree :)

**Comment by AndyButland** [comment]
Looks good to me, thanks.
